packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -19,7 +19,7 @@
 class NodeCreate(InputSchemaWithoutCamelCase):
     service_key: ServiceKey
     service_version: ServiceVersion
-    service_id: str | None
+    service_id: str | None = Field(default=None)
```

Comment:
this is also a change in syntax.

Reply:
@pcrespov ?

Reply:
OpenAPI Spec says it's not required. I consider any failing test as error in the test itself

Reply:
OpenAPI Spec (3.0.1) - master branch:

```yaml
NodeCreate:
      title: NodeCreate
      required:
      - service_key
      - service_version
      type: object
      properties:
        service_key:
          title: Service Key
          pattern: ^simcore/services/((comp|dynamic|frontend))/([a-z0-9][a-z0-9_.-]*/)*([a-z0-9-_]+[a-z0-9])$
          type: string
        service_version:
          title: Service Version
          pattern: ^(0|[1-9]\d*)(\.(0|[1-9]\d*)){2}(-(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*)(\.(0|[1-9]\d*|\d*[-a-zA-Z][-\da-zA-Z]*))*)?(\+[-\da-zA-Z]+(\.[-\da-zA-Z-]+)*)?$
          type: string
        service_id:
          title: Service Id
          type: string
```
I use this a source of truth, and consider any failing test as an error in the test itself.